packages/manager/modules/pci/src/projects/project/training/jobs/translations/Messages_fr_FR.json

```
@@ -2,6 +2,7 @@
   "pci_projects_project_training_jobs_list_message_1": "Pour accéder à cette liste de jobs via la ligne de commande, lancez la commande suivante :",
   "pci_projects_project_training_jobs_submit_select_region": "Choisir une région",
   "pci_projects_project_training_jobs_submit_select_image": "Renseigner l'image Docker",
+  "pci_projects_project_training_jobs_submit_select_image_name": "Nom de l'image docker",
```

```suggestion
  "pci_projects_project_training_jobs_submit_select_image_name": "Nom de l'image Docker",
```

see: https://www.docker.com/legal/trademark-guidelines (Use of capitals)

---

Done